scripts/hook.py

```diff
@@ -0,0 +1,137 @@
+import requests
```

Comment:
can you please replace the name and the `400` number throughout this with something else? I realise this was given to you as test data but if this is going to be committed into the history of this project (forever) I'd prefer it was a little more anonymous.